recipe/meta.yaml

```
@@ -15,16 +15,18 @@ source:
 
 build:
   number: 0
-  run_exports:   # [cross_target_platform == "osx-64"]
-    strong_constrains:   # [cross_target_platform == "osx-64"]
-      - __osx>=10.12    # [cross_target_platform == "osx-64"]
-  run_exports:   # [cross_target_platform == "linux-64" and go_variant_str == "cgo"]
-    strong:   # [cross_target_platform == "linux-64" and go_variant_str == "cgo"]
-      - __glibc >=2.17                    # [cross_target_platform == "linux-64" and go_variant_str == "cgo"]
+  skip: True            # [s390x]
+  run_exports:          # [osx]
+    strong_constrains:  # [osx]
+      - __osx>=10.12    # [osx]
+  run_exports:          # [linux and go_variant_str == "cgo"]
```

```suggestion
  run_exports:          # [linux and go_variant_str == "cgo"]
  run_exports:          # [linux and cgo]
```
Is it possible to just do something like this since the go_variant_str is in the cbc?

---

The values added to the cbc were `go_compiler` and `go_compiler_version`.